golos.charge/golos.charge.hpp

```diff
@@ -115,7 +116,7 @@ class charge : public contract {
             eosio_assert(restored >= fixp_t(0), "charge::calc_value restored < 0");
         }
         
-        return std::max(fp_cast<fixp_t>((elap_t(FP(user_balance.value)) - elap_t(restored)) + elap_t(price)), fixp_t(0));
+        return std::max(FP(user_balance.value) - fp_cast<fixp_t>(elap_t(restored) * elai_t(config::_100percent)) + price, fixp_t(0));
```

Comment:
a little confusing, that restored is measured in other units, but in this sense, nothing has changed. just need to keep it in mind when setting restorer.

Reply:
@kaynarov, don't this cuts accuracy of `restored`?

and it looks that price can be fully compensated by over-restoring.
e.g. if we first "saturate" negatives to `0` and than add price, then the usage will be equal to price. but current implementation first adds price, and than saturate to 0